components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -293,6 +293,8 @@ - (void)setHighlightPoint:(CGPoint)highlightPoint {
 }
 
 - (void)layoutSubviews {
+  UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification,
+                                  self.accessibilityLabel);
```

Comment:
Won't this get invoked every time there we layout frames?
Would this make more send to invoke one on the VC's viewDidAppear: ?

Reply:
Done